src/views/Access/Employees.tsx

```diff
@@ -1,5 +1,71 @@
-import { Heading } from '@chakra-ui/react';
+import { Image, Grid, GridItem, Box } from '@chakra-ui/react';
+import Supplier from 'views/Access/Supplier';
+import Waiter from 'views/Access/Waiter';
+import Chef from 'views/Access/Chef';
+import Businessman from 'views/Access/Businessman';
+import Driver from 'views/Access/Driver';
+import supplier from 'assets/images/supplier.png';
+import waiter from 'assets/images/waiter.png';
+import chef from 'assets/images/chef.png';
+import businessman from 'assets/images/businessman.png';
+import driver from 'assets/images/driver.png';
```

Comment:
As in my previous comment.
In my opinion `chakra-ui/icons` and `react-icons` provide all required icons. You don'y need to import images / icons from external sources.
We should not mix different sets.

Reply:
But it's not an icon, it's a typical graphic. In the available resources chakra-ui/icons and react-icons I couldn't find suitable icons for these graphics

![image](https://user-images.githubusercontent.com/43721352/123558221-5f114b80-d795-11eb-9743-bf5071f590ac.png)